submit-web/src/components/Submission/ItemsTable/index.tsx

```diff
@@ -45,7 +45,7 @@ export default function ItemsTable({ submissionPackage }: ItemsTableProps) {
             />
           ))}
           <When condition={userType === USER_TYPE.STAFF}>
-            <InternalDocumentsRows numColumns={5} hideAction />
```

Comment:
this is a change from general and can be used anywhere to logic that is tied to a page